vm/systemSmartContracts/validator.go

```diff
@@ -1340,6 +1384,27 @@ func (v *validatorSC) basicCheckForUnStakeUnBond(args *vmcommon.ContractCallInpu
 	return registrationData, vmcommon.Ok
 }
 
+func (v *validatorSC) getUnStakedTokensList(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
no unit test. Should have least added or changed a Jira task. I have updated EN-8525

Reply:
Thx for the update